Rasa_Bot/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Pull SDK image as base image
+FROM rasa/rasa-sdk:2.0.0
+
+# Use subdirectory as working directory
+WORKDIR /app
+
+# Copy actions requirements
+# COPY actions/requirements-actions.txt ./
+
+# Change to root user to install dependencies
+USER root
+
+# Install extra requirements for actions code
+# RUN pip install -r requirements-actions.txt
```

Comment:
I suppose these relate to later work on the actions server (i.e. a later PR?)

Reply:
Yes, these are in case we need to install dependencies for the custom actions in the future. This could be things like pandas, etc.